src/messages/text_document_completion.cc

```diff
@@ -365,7 +365,7 @@ struct Handler_TextDocumentCompletion : MessageHandler {
             lock.lock();
           std::string quote = result.match[5];
           for (auto& item : include_complete->completion_items)
-            if (quote.empty() || quote == (item.use_angle_brackets_ ? "<" : "\""))
+            // if (quote.empty() || quote == (item.use_angle_brackets_ ? "<" : "\""))
```

Comment:
I don't remember what this is about.

`-I` available in both `#include "` and `#include <`
`-isystem` available in `#include <`
`-iquote` available in `#include "`

Reply:
I see. `-I` now adds dir to both categories.